src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
```

Comment:
- [ ] After importing `categoriesSlice`, please add it to the reducers here. It can look like this
```
  reducer: {
    books: booksReducer,
    categories: categoriesReducer,
  },
```